source/user-manual/capabilities/system-inventory/compatibility-matrix.rst

```diff
@@ -25,3 +25,29 @@ The Syscollector module supports different options across various operating syst
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
 |    Solaris             |     ✗     |     √     |     √     |     √    |     ✗     |     ✗     |     –     |
 +------------------------+-----------+-----------+-----------+----------+-----------+-----------+-----------+
+
+.. versionadded:: 4.7.0
+
+The Syscollector module now supports scan packages from `NPM (Node Package Manager)  <https://www.npmjs.com/>`_ and `PyPI (Python Package Index)  <https://pypi.org/>`_, **the support is only for the default paths of those packages' installation**.
+
+The following table shows the new feature that are compatible with various operating systems.
```

Comment:
If we are talking about "the new feature" I think it should be "...is compatible..."

Reply:
I agree :)